.chloggen/telemetrygen-traces-update.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: telemetrygen
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Adds options to configure the use of the batchspanprocessor and the size in MB of each trace for load testing.
+
+# One or more tracking issues related to the change
+issues: [9597]
+
+# (Optional) One or more lines of additional information to render under the primary note.
+# These lines will be padded with 2 spaces and then inserted directly into the document.
+# Use pipe (|) for multiline entries.
+subtext: Fixes issue where service name option didn't properly set on the trace, and fixes issue where large trace payloads (>2000) wouldn't generate the correct number of traces.
```

Comment:
Instead of listing several fixes here let's split into several changelog entries in separate files